src/Microsoft.ApplicationInsights/OperationTelemetryExtensions.cs

```diff
@@ -33,7 +38,7 @@ public static void Start(this OperationTelemetry telemetry)
         /// <param name="timestamp">A high-resolution timestamp from <see cref="Stopwatch"/>.</param>
         public static void Start(this OperationTelemetry telemetry, long timestamp)
         {
-            telemetry.Timestamp = DateTimeOffset.UtcNow;
+            telemetry.Timestamp = DateTimeOffset.UtcNow; 
```

Comment:
Shouldn't this also use the PreciseTimestamp ?

Reply:
good catch! this was part of my testing, forgot to return it back